### PR TITLE
Add initial tags based on docs organization.

### PR DESCRIPTION
**Pull request by andrewsomething**

### CODE REVIEW

File: DigitalOcean-public.v2.yaml [hunk 1]
```diff
@@ -13,6 +13,11 @@ info:
 servers:
   - url: 'https://api.digitalocean.com/v2'
     description: production
+tags:
+  - name: account
+    description: Account
```
Comment (hilary) [inline DigitalOcean-public.v2.yaml]: We should either leave the description out or provide more than a one-word description that is the same as the tag....

File: DigitalOcean-public.v2.yaml [hunk 2]
```diff
@@ -13,6 +13,11 @@ info:
...
+  - name: ssh_keys
+    description: SSH Keys
```
Comment (hilary) [inline DigitalOcean-public.v2.yaml]: So, for example, this description can explain that it is the resource name and our thinking around resources.

**Review by hilary — CHANGES_REQUESTED** [review]

File: DigitalOcean-public.v2.yaml (hunk 1 above)
Comment (andrewsomething) [reply DigitalOcean-public.v2.yaml]: ah, plugging it into Swagger Hub I better see how these are getting used. 

image

Updated

**Review by hilary — APPROVED** [review]
:shipit:

**Pull request merged**